servicetest/timingprofiling/test_profiling.py

```diff
@@ -145,9 +145,9 @@ def get_function_log(log_file, function_name, match_number=1):
         if re.search(function_name + " end", line):
             match = match + 1
             if match == match_number:
-                print line
+                print(line)
                 return remove_ansi(string.split(line)[6])
-    print "Nothing found! " + function_name + " " + str(match_number)
+    print("Nothing found! " + function_name + " " + str(match_number))
```

Comment:
Use .format instead of +
https://softwareengineering.stackexchange.com/questions/304445/why-is-s-better-than-for-concatenation